docs/resources/rdb_read_replica.md

```diff
@@ -65,8 +65,9 @@ The following arguments are supported:
 - `direct_access` - (Optional) Creates a direct access endpoint to rdb replica.
 
 - `private_network` - (Optional) Create an endpoint in a private network.
-    - `private_network_id` - (Required) UUID of the private network to be connected to the read replica.
-    - `service_ip` - (Required) Endpoint IPv4 address with a CIDR notation. Check documentation about IP and subnet
+    - `private_network_id` - (Required) UUID of the private network to be connected to the read replica. Handled by IPAM
+      is not set.
```

Comment:
Maybe review the doc format and the wording

Reply:
doc reworked